tf2_eigen/package.xml

```diff
@@ -1,20 +1,26 @@
 <?xml version="1.0"?>
```

Comment:
Tully merged that into the ros/geometry2 repository, so we can merge up with that to get those fixes in here.  I'll do that once the review here is done.